metainfo.xml

```diff
@@ -209,6 +209,13 @@
         </osSpecific>
       </osSpecifics>
 
+      <themes>
+        <theme>
+          <fileName>theme.json</fileName>
```

Comment:
Nope. Ambari expects all theme files to be in "themes" already.

Example:
https://github.com/apache/ambari/blob/trunk/ambari-server/src/main/resources/stacks/HDP/2.2/services/HBASE/metainfo.xml#L51-L56